sms/src/main/java/org/motechproject/sms/web/StatusController.java

```diff
@@ -212,7 +212,7 @@ private void analyzeStatus(Status status, String configName, Map<String, String>
                     configName, params);
             LOGGER.error(msg);
             statusMessageService.warn(msg, SMS_MODULE);
-            smsRecord.setDeliveryStatus(DeliveryStatus.FAILURE_CONFIRMED);
+            smsRecord.setDeliveryStatus(DeliveryStatuses.FAILURE_CONFIRMED);
```

Comment:
string status is null here, should we use constant or status.getStatusFailure() value?